playground/src/modules/x/counter/counter.js

```diff
@@ -0,0 +1,12 @@
+import { LightningElement } from "lwc";
```

Comment:
I mentioned formatting earlier because I noticed the `"` here. I'm actually amazed I even noticed this, I guess it's because we've been formatting our code base for so long!

Reply:
Fixed